docs/projects.md

```diff
@@ -278,3 +281,96 @@ You should see something like this:
 ```
 
 And, most importantly, your browser should open a page with a jupyter notebook.
+
+
+## Using Github
+There are three main ways to access a local copy of the Class Code, which is stored in the Github repository.  The first way is to navigate to the class Github repository click on the green code button and select 'Download Zip'.  This will provide you with a ZIP file of the class code, which can be run on your personal machine.  Now you might be saying, that is great but I don't want to download a new ZIP file every time something changes.  In that case you want a local copy of the class Github repository.  You should use on the the two following methods to accomplish this.
+
+### Using Github with a Clone
+A clone is a copy of a repository.  You are free to make changes to the repository and will be able to get an up to date version of the original repository.
+To clone the class repository simply click the green **Code** button in the header of the repository and copy the url presented to you for cloning ```https://github.com/tbretl/ae353-sp21.git```. Make sure you have Git installed on your computer (if not view instructions below under Install Git client tools) and then git bash into the directory where you want to store the local copy of the repository.  Then run the following commands.
```

Comment:
Add comma after "if not" in the parentheses. Change last word to "command" since there is only one command to run.